File: day-1/part-one.py
with open('C:/Users/HP/Desktop/advent-of-code-2023/day-1/puzzle-input.txt', 'r', encoding='utf-8') as f:
    list_of_calibration_values = []
    for line in f:
        inner_values = []
        for i in line:
            if i.isdigit():
                inner_values.append(i)
        value = inner_values[0] + inner_values[-1]
        int_value = int(value)
        list_of_calibration_values.append(int_value)
calibration_values = sum(list_of_calibration_values)
print(calibration_values)
            
        

File: 2023/solution-in-javascript/day-1/part-one.js
const fs = require('fs');

const filePath = 'C:/Users/HP/Desktop/advent-of-code-solution/2023/puzzle-input/day2.txt';
const fileContent = fs.readFileSync(filePath, 'utf-8');
const lines = fileContent.split('\n');

for(;;){
    console.log(lines);
}

// let list_of_calibration_values = [];

// lines.forEach((line) => {
//   const inner_values = [];
//   for (let i = 0; i < line.length; i++) {
//     if (!isNaN(parseInt(line[i]))) {
//       inner_values.push(line[i]);
//     }
//   }
//   const value = inner_values[0] + inner_values[inner_values.length - 1];
//   const int_value = parseInt(value);
//   list_of_calibration_values.push(int_value);
// });

// const calibration_values = list_of_calibration_values.reduce((acc, val) => acc + val, 0);
// console.log(calibration_values);
